R-Packages.js

```diff
@@ -125,26 +170,255 @@ var testCases = [
+		"items": [
+			{
+				"itemType": "computerProgram",
+				"title": "effects: Effect Displays for Linear, Generalized Linear, and Other Models",
+				"creators": [
+					{
+						"firstName": "John",
+						"lastName": "Fox",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Sanford",
+						"lastName": "Weisberg",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Michael",
+						"lastName": "Friendly",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Jangman",
+						"lastName": "Hong",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Robert",
+						"lastName": "Andersen",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "David",
+						"lastName": "Firth",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Steve",
+						"lastName": "Taylor",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "R. Core",
+						"lastName": "Team",
+						"creatorType": "author"
 					}
 				],
-				"date": "2016-10-05",
-				"abstractNote": "Graphical and tabular effect displays, e.g., of interactions, for \n  various statistical models with linear predictors.",
+				"date": "2017-09-15",
+				"abstractNote": "Graphical and tabular effect displays, e.g., of interactions, for various statistical models with linear predictors.",
 				"libraryCatalog": "R-Packages",
 				"rights": "GPL-2 | GPL-3 [expanded from: GPL (≥ 2)]",
 				"shortTitle": "effects",
-				"url": "http://cran.us.r-project.org/web/packages/effects/index.html",
-				"version": "3.1-2",
-				"attachments": [
+				"url": "https://CRAN.R-project.org/package=effects",
+				"versionNumber": "4.0-0",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Econometrics"
+					},
+					{
+						"tag": "MachineLearning"
+					},
+					{
+						"tag": "SocialSciences"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://cloud.r-project.org/web/packages/effects/index.html",
+		"items": [
+			{
+				"itemType": "computerProgram",
+				"title": "effects: Effect Displays for Linear, Generalized Linear, and Other Models",
+				"creators": [
+					{
+						"firstName": "John",
+						"lastName": "Fox",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Sanford",
+						"lastName": "Weisberg",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Michael",
+						"lastName": "Friendly",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Jangman",
+						"lastName": "Hong",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Robert",
+						"lastName": "Andersen",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "David",
+						"lastName": "Firth",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Steve",
+						"lastName": "Taylor",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "R. Core",
```

Comment:
we should recognize the R Core Team and turn it into an institutional author. Presumably that'll show up on a fair number of packages, so should be worth it.

Reply:
Done in the new version.